composer.json

```diff
@@ -13,7 +13,7 @@
     "optimize-autoloader": true,
     "classmap-authoritative": false,
     "platform": {
-      "php": "7.0.8"
+      "php": "7.2"
```

Comment:
Note: for composer 2.0 ... we need PHP 7.2. That should be OK, because future releases of `files_primary_s3` will be for core 10.5 onwards anyway.

Reply:
Actually we can still use PHP 7.1 here. I changed it to 7.1 - that matches with what is already in `appinfo/info.xml` anyway.